conf/routes

```diff
@@ -60,4 +60,8 @@ GET         /.well-known/security.txt                                controllers
 GET         /sitemap-documentation.xml                               controllers.documentation.DocumentationController.sitemap
 GET         /robots.txt                                              controllers.Application.robots
 
+GET         /blog                                                    controllers.Blog.index
+GET         /play-on-graal/                                          controllers.Blog.graal
+GET         /play-socket-io/                                         controllers.Blog.socketio
```

Comment:
the current URL for these two pages is:

https://blog.playframework.com/play-on-graal/

But moving them into the Play application (and fixing the DNS entry for `blog.playframework.com`) I would change that to:


Request: GET https://blog.playframework.com/play-on-graal/ --> 
Responds: 301 Moved Permanently https://www.playframework.com/blog/play-on-graal/

That is: stop using `blog.play.com` as a subdomain and standardize using all blog content under `/blog/.*`

Reply:
addressed in [`bc9f809` (#421)](https://github.com/playframework/playframework.com/pull/421/commits/bc9f809c5f2a10e04644707c1b242ac6a542f76f)